src/renderer40k.ts

```diff
@@ -193,6 +196,34 @@ export class Renderer40k implements Renderer {
                     detachmentSheets.classList.remove('d-none')
                 }
             });
+        // ability to hide divs (abilities, rules, ...) from printing
+        optionsDiv.appendChild(document.createElement('span')).innerHTML = '&nbsp;&nbsp;&nbsp;&nbsp;';
+        this.renderCheckboxOption(optionsDiv, 'hideElements', 'Hide Elements from Printing',
+            (e: Event) => {
+                if ((e.target as any).checked) {
+                    const customize = document.createElement('style');
```

Comment:
consider moving this styling to the CSS; you'll still need to toggle something in the DOM, but you can toggle a class either on the body or individual hideable elements, and then the hover rule will have to be gated off that class as well. The benefit is to consolidate all CSS in a central location